test/compress/issue-2001.js

```diff
@@ -61,3 +61,61 @@ export_default_func_3: {
     }
     expect_exact: "export default function(){};"
 }
+
+export_mangle_1: {
+    mangle = {}
+    input: {
+        export function foo(one, two) {
+            return one - two;
+        };
+    }
+    expect_exact: "export function foo(n,o){return n-o};"
+}
+
+export_mangle_2: {
+    mangle = {}
+    input: {
+        export default function foo(one, two) {
+            return one - two;
+        };
+    }
+    expect_exact: "export default function foo(n,o){return n-o};"
+}
+
+export_mangle_3: {
+    mangle = {}
+    input: {
+        export class C {
+            go(one, two) {
+                var z = one;
+                return one - two + z;
+            }
+        };
+    }
+    expect_exact: "export class C{go(n,r){var t=n;return n-r+t}};"
+}
+
+export_mangle_4: {
+    mangle = {}
+    input: {
+        export default class C {
+            go(one, two) {
+                var z = one;
+                return one - two + z;
+            }
+        };
+    }
+    expect_exact: "export default class C{go(n,r){var t=n;return n-r+t}};"
```

Comment:
Can you enable `collapse_vars` in this test to also test `compress` eliminating `t`?